src/views/HabitPage/components/HabitMain.tsx

```
@@ -0,0 +1,62 @@
+
+import formatDateTime from "@/shared/hooks/Time";
+import { useState, useEffect } from "react";
+import {
+  ButtonStart,
+  ButtonStartDisabled,
+} from "@/shared/components/button/ButtonRound";
+import { ButtonHome } from "@/shared/components/button/ButtonNav";
+
+export default function HabitMain() {
+  const [currentTime, setCurrentTime] = useState(formatDateTime);
+  const [isPaused, setIsPaused] = useState(false);
```

ESLint

[error] 'isPaused' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)